### PR TITLE
Adding some missing entries to the changelog.

### PR DESCRIPTION
**Pull request by garethgreenaway**
### What does this PR do?
Adding missing PRs into the change log for Sodium

### Merge requirements satisfied?
**[NOTICE] Bug fixes or features added to Salt require tests.**

- [ ] Docs
- [x] Changelog - https://docs.saltstack.com/en/master/topics/development/changelog.html
- [ ] Tests written/updated

### Commits signed with GPG?
Yes

Please review [Salt's Contributing Guide](https://docs.saltstack.com/en/master/topics/development/contributing.html) for best practices.

See GitHub's [page on GPG signing](https://help.github.com/articles/signing-commits-using-gpg/) for more information about signing commits with GPG.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -52,6 +53,11 @@ Changed
 - Upgraded dependency to use boto3>=1.13.5. (#57161)
 - Changed to consistent file location handling across APIs for Juniper network devices. (#57399)
 - Use Python's hashlib (sha256) instead of shelling out (SipHash24) to generate server_id. (#57415)
+- Some Debian network (debian_ip) refactoring (#49462)
+- Port #53537 to master (#56835)
```
Comment (waynew) [inline CHANGELOG.md:57]: This is already in here - I'm suspecting the other ones will be as well - https://github.com/saltstack/salt/pull/53537 fixes issue https://github.com/saltstack/salt/issues/4438

**Review by waynew — CHANGES_REQUESTED** [review]

**Review by DmitryKuzmenko — DISMISSED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -27,6 +27,7 @@ Removed
 - Deprecated `refresh_db` removed from pkgrepo state. Use `refresh` instead. (#57366)
 - Deprecated internal functions salt.utils.locales.sdecode and .sdecode_if_string removed. Use salt.utils.data.decode instead. (#57367)
 - Removed deprecated misc. internal Salt functions. See https://github.com/saltstack/salt/issues/57368 for more info. (#57368)
+- Remove salt/utils/vt.py duplication from filename map. (#57004)
```
Comment (waynew) [inline CHANGELOG.md:30]: This isn't a feature removal - also it's only in the `tests` dir, so really not user-facing.

File: CHANGELOG.md [hunk 3]
```diff
@@ -54,6 +55,7 @@ Changed
 - Upgraded dependency to use boto3>=1.13.5. (#57161)
 - Changed to consistent file location handling across APIs for Juniper network devices. (#57399)
 - Use Python's hashlib (sha256) instead of shelling out (SipHash24) to generate server_id. (#57415)
+- Update `formulas.rst` with new IRC channel and links to IRC logs (#51628)
```
Comment (waynew) [inline CHANGELOG.md:58]: I don't feel strongly that this *shouldn't* be added, but I don't feel strongly that it should be added either.

I guess I have a weak preference that it doesn't /really/ belong in the changelog, in that I would be shocked if someone was led to the correct IRC channel by encountering the entry here in the changelog.

File: CHANGELOG.md [hunk 4]
```diff
@@ -262,6 +264,11 @@ Fixed
 - Fixed `win_system.join_domain` failures. (#57360)
 - Fixed `template_vars` functionality on Junos. (#57388)
 - Filter out aliases/duplicates from zypperpkg for <=SLE12SP4. (#57392)
+- Wrong isinstance in ini options absent (#53874)
```
Comment (waynew) [inline CHANGELOG.md:267]: https://github.com/saltstack/salt/blame/master/CHANGELOG.md#L134 already there

File: CHANGELOG.md [hunk 5]
```diff
@@ -262,6 +264,11 @@ Fixed
...
+- Fix issue with finding the real python executable during tests (#56460)
```
Comment (waynew) [inline CHANGELOG.md:268]: https://github.com/saltstack/salt/pull/56460 wrong #?

File: CHANGELOG.md [hunk 6]
```diff
@@ -262,6 +264,11 @@ Fixed
...
+- Fix broken link regarding the 1024 character limit for YAML keys (#56540)
```
Comment (waynew) [inline CHANGELOG.md:269]: another doc change that seems low impact.

File: CHANGELOG.md [hunk 7]
```diff
@@ -262,6 +264,11 @@ Fixed
...
+- Fix grain.delkey grains.delval for nested keys (#54819)
+- Virt disks fixes (#56666)
```
Comment (waynew) [inline CHANGELOG.md:271]: Already exists as https://github.com/saltstack/salt/blame/master/CHANGELOG.md#L353

**Review by waynew — CHANGES_REQUESTED** [review]
A couple of these entries are already in the changelog

**Review by DmitryKuzmenko — APPROVED** [review]

**Review by waynew — APPROVED** [review]

**Pull request merged**